scipy/optimize/minpack.py

```diff
@@ -292,7 +292,7 @@ def leastsq(func, x0, args=(), Dfun=None, full_output=0,
     func : callable
         Should take at least one (possibly length N vector) argument and
         returns M floating point numbers. It must not return NaNs or
-        fitting might fail.
+        fitting might fail. M must be greater than or equal to N (M >= N).
```

Comment:
Don't need the "equation" for this simple text.
```suggestion
        fitting might fail. ``M`` must be greater than or equal to ``N``.
```